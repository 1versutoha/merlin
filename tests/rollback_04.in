["tell","source","module X = struct\n  List\nend\n"]
["tell","end"]
["seek","before",{"line":2,"col":0}]
["tell","source","\n  include List\nend\n"]
["tell","end"]
["errors"]
