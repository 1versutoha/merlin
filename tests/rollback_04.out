["return",null]
["return",{"line":3,"col":3}]
["return",{"line":1,"col":17}]
["return",null]
["return",{"line":3,"col":3}]
["return",[]]
